### ISSUE TITLE
WARNING: custom_target '*' has more than one output! Using the first one.

### ISSUE THREAD

**Issue by phrdina** [mesonbuild/meson#7466]
**Describe the bug**
When there is a custom_target() with more then one output we will get a warning printed to stdout:

WARNING: custom_target 'gen_empty_files' has more than one output! Using the first one.

**To Reproduce**
```
project('reproduce', ['c'], meson_version: '>=0.55.0')
  
custom_target(
  'gen_empty_files',
  output: [ 'file1', 'file2', 'file3' ],
  command: [ 'touch', '@OUTPUT@' ],
)
```

git bisect points to this commit https://github.com/mesonbuild/meson/commit/b4b1a2c5a145c1459fc4563a289e164e23bd6a02

**Expected behavior**
No warning printed to stdout.

**system parameters**
* Is this a [cross build](https://mesonbuild.com/Cross-compilation.html) or just a plain native build (for the same computer)?
native build
* what operating system (e.g. MacOS Catalina, Windows 10, CentOS 8.0, Ubuntu 18.04, etc.)
Fedora 32
* what Python version are you using e.g. 3.8.0
Python 3.8.3
* what `meson --version`
0.55.0
* what `ninja --version` if it's a Ninja build
1.10.0

**Comment by keszybz** [comment]
Should be fixed by 1123f4f311e1ac89b341dea2cb5922f29dc502e3.

**Comment by nirbheek** [comment]
Yes, fixed by https://github.com/mesonbuild/meson/pull/7494. Marked for 0.55.1 already.
